src/routes/github/configuration/github-configuration-get.ts

```diff
@@ -137,10 +154,6 @@ export const GithubConfigurationGet = async (req: Request, res: Response, next:
 
 	const log = req.log.child({ jiraHost });
 
-	if (!githubToken) {
```

Comment:
Why aren't these protected by FF?

Reply:
Because this was some old dead code. We always have gitHubToken here because it is under GitHubOauthMiddelware